presto-parser/src/main/java/com/facebook/presto/sql/tree/Identifier.java

```diff
@@ -87,12 +87,12 @@ public boolean equals(Object o)
         }
 
         Identifier that = (Identifier) o;
-        return Objects.equals(value, that.value);
```

Comment:
Yes, but we use equality to not only to determine "equivalency in all cases" but also to determine "equivalency in this or that particular case". Like in determining column usage when there is grouping. Or for handling `grouping` operation, etc